controls/V-72299.rb

```diff
@@ -1,57 +1,40 @@
-Status: Accepted
-
-This Security Technical Implementation Guide is published as a tool to improve
-the security of Department of Defense (DoD) information systems. The
-requirements are derived from the National Institute of Standards and
-Technology (NIST) 800-53 and related documents. Comments or proposed revisions
-to this document should be sent via email to the following address:
-disa.stig_spt@mail.mil.
-
-Release Date: 2017-03-08
-Version: 1
-Publisher: DISA
-Source: STIG.DOD.MIL
-uri: http://iase.disa.mil
------------------
-=end
-
 control "V-72299" do
-  title "A File Transfer Protocol (FTP) server package must not be installed unless
-needed."
-  desc  "The FTP service provides an unencrypted remote access that does not provide
-for the confidentiality and integrity of user passwords or the remote session. If a
-privileged user were to log on using this service, the privileged user password
-could be compromised. SSH or other encrypted file transfer methods must be used in
-place of this service."
+  title "A File Transfer Protocol (FTP) server package must not be installed
+unless needed."
+  desc  "The FTP service provides an unencrypted remote access that does not
+provide for the confidentiality and integrity of user passwords or the remote
+session. If a privileged user were to log on using this service, the privileged
+user password could be compromised. SSH or other encrypted file transfer
+methods must be used in place of this service."
   impact 0.7
-  tag "severity": "high"
   tag "gtitle": "SRG-OS-000480-GPOS-00227"
   tag "gid": "V-72299"
-  tag "rid": "SV-86923r1_rule"
+  tag "rid": "SV-86923r2_rule"
   tag "stig_id": "RHEL-07-040690"
-  tag "cci": "CCI-000366"
+  tag "cci": ["CCI-000366"]
+  tag "documentable": false
   tag "nist": ["CM-6 b", "Rev_4"]
-  tag "check": "Verify a lightweight FTP server has not been installed on the system.
+  tag "check": "Verify an FTP server has not been installed on the system.
+
+Check to see if an FTP server has been installed with the following commands:
 
-Check to see if a lightweight FTP server has been installed with the following
-commands:
+# yum list installed vsftpd
 
-# yum list installed lftpd
- lftp-4.4.8-7.el7.x86_64.rpm
+ vsftpd-3.0.2.el7.x86_64.rpm
 
-If \"lftpd\" is installed and is not documented with the Information System Security
-Officer (ISSO) as an operational requirement, this is a finding."
-  tag "fix": "Document the \"lftpd\" package with the ISSO as an operational
+If \"vsftpd\" is installed and is not documented with the Information System
+Security Officer (ISSO) as an operational requirement, this is a finding.
+"
+  tag "fix": "Document the \"vsftpd\" package with the ISSO as an operational
 requirement or remove it from the system with the following command:
 
-# yum remove lftpd"
+# yum remove vsftpd
+"
+  tag "fix_id": "F-78653r2_fix"
 
-  describe package('lftpd') do
+  describe package('vsftpd') do
```

Comment:
Per the prose, this should only apply if `vsftpd` is not set up for TLS connections. FTPS should be just fine since the justification for removal is that a "FTP service provides unencrypted remote access".